### PR TITLE
Common rule for import's order and parser

### PR DESCRIPTION
**Pull request by RajaRamu**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,3 +1,13 @@
 {
-    "extends": "ebay"
-}
\ No newline at end of file
+    "extends": "ebay",
+    "rules": {
+         "import/order": [2, {"groups": ["builtin", "external", "internal", "parent", "sibling", "index"]}],
+        "no-param-reassign": 0
```
Comment (mikewoo200) [inline .eslintrc]: I think that `no-param-reassign` (currently set as `2`) in keeping a parameter immutable is helpful in preventing bugs. The current rule does allow object property to be reassigned.

https://github.com/eBay/eslint-config-ebay/blob/master/rules/bp.js#L90

https://github.com/eBay/eslint-config-ebay/commit/bb4b09d7b869251cd958b844ade6130eff2d9d70

File: .eslintrc [hunk 2]
```diff
@@ -1,3 +1,13 @@
 {
-    "extends": "ebay"
-}
\ No newline at end of file
+    "extends": "ebay",
+    "rules": {
+         "import/order": [2, {"groups": ["builtin", "external", "internal", "parent", "sibling", "index"]}],
```
Comment (mikewoo200) [inline .eslintrc]: For `import/order`, do we have to add this as a dev dependency?

https://github.com/benmosher/eslint-plugin-import

File: .eslintrc [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+        "no-param-reassign": 0
+    },
+    "parserOptions": {
+        "ecmaVersion": 2017,
```
Comment (mikewoo200) [inline .eslintrc]: `eslint-config-ebay` is currently on version `2.x`. 

ecmaVersion native support for `2017` requires `eslint@3.6.0` or higher

https://eslint.org/blog/2016/09/eslint-v3.6.0-released

We should probably consider upgrading `eslint` version itself.

Comment (mikewoo200) [inline .eslintrc]: Note to self: Changes in this PR will require a major version change to `^1.0.0`

Comment (mikewoo200) [reply .eslintrc]: Also, we reference `"ecmaVersion": 2017` here as well. We can probably just keep it in one place.

https://github.com/eBay/eslint-config-ebay/blob/master/index.js#L8-L10

File: .eslintrc (hunk 2 above)
Comment (CestDiego) [reply .eslintrc]: I believe that's the case. Yes

File: .eslintrc (hunk 1 above)
Comment (CestDiego) [reply .eslintrc]: I agree with this. We should keep it as such.  Also makes it easier to have a more functional approach

**Comment by CestDiego** [comment]
Is there a way to just add the import rule without the rest of rules? @RajaRamu

**Comment by RajaRamu** [comment]
@CestDiego - I will refresh this PR shortly.

File: .eslintrc (hunk 2 above)
Comment (RajaRamu) [reply .eslintrc]: Yes @CestDiego & @mikewoo200  . Lets come with order if you have any thought

**Comment by RajaRamu** [comment]
Closing this and will open new PR with wide consultation with folks

**Pull request closed without merging**